data/json/items/tool/science.json

```diff
@@ -463,7 +463,7 @@
     "id": "filter_paper_qualitative",
     "type": "TOOL",
     "name": { "str_sp": "qualitative filter paper" },
-    "description": "filter paper for seperating solids from liquids.  This filter paper has a rating of 2 microns, seperating more solids out than quantitative filter paper but the process takes longer.",
+    "description": "filter paper for separating solids from liquids.  This filter paper has a rating of 2 microns, separating more solids out than quantitative filter paper but the process takes longer.",
```

Comment:
Could you please capitalize the first word in the sentence while you're here?

Reply:
Yeap